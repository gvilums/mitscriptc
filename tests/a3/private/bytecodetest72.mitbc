function
{
	functions =
	[
		function
		{
			functions = [],
			constants = [],
			parameter_count = 1,
			local_vars = [],
			local_ref_vars = [],
			free_vars = [],
			names = [],
			instructions = 
			[
			]
		},
		function
		{
			functions = [],
			constants = [],
			parameter_count = 0,
			local_vars = [],
			local_ref_vars = [],
			free_vars = [],
			names = [],
			instructions = 
			[
			]
		},
		function
		{
			functions = [],
			constants = [],
			parameter_count = 1,
			local_vars = [],
			local_ref_vars = [],
			free_vars = [],
			names = [],
			instructions = 
			[
			]
		}
	],
	constants = [1, "made it here"],
	parameter_count = 0,
	local_vars = [],
	local_ref_vars = [],
	free_vars = [],
	names = [z, print, input, intcast],
	instructions = 
	[
		load_func	0
		alloc_closure	0
		store_global	1
		load_func	1
		alloc_closure	0
		store_global	2
		load_func	2
		alloc_closure	0
		store_global	3
		load_const	0
		store_global	0
		load_global	1
		load_global	0
		call	1
		pop
		load_global	0
		not
		if	5
		load_global	1
		load_const	1
		call	1
		pop
	]
}