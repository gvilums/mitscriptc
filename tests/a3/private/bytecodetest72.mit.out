1
IllegalCastException: Uninitialized to Bool
